apps/federation/lib/TrustedServers.php

```diff
@@ -98,13 +98,32 @@ public function removeServer(int $id): void {
 	 * @return list<array{id: int, url: string, url_hash: string, shared_secret: ?string, status: int, sync_token: ?string}>
 	 * @throws Exception
 	 */
-	public function getServers() {
+	public function getServers(): ?array {
 		if ($this->trustedServersCache === null) {
 			$this->trustedServersCache = $this->dbHandler->getAllServer();
 		}
 		return $this->trustedServersCache;
 	}
 
+	/**
+	 * Get a trusted server
+	 *
+	 * @return array{id: int, url: string, url_hash: string, shared_secret: ?string, status: int, sync_token: ?string}
+	 * @throws Exception
```

Comment:
```suggestion
	 * @throws \Exception
```
`Exception` here refers to \OC\DB\Exception which is in the use statements, but you are throwing `\Exception`.

Reply:
done